pkg/sqlqueries/SQL/system_intake_system/select_by_cedar_system_id.sql

```diff
@@ -0,0 +1,8 @@
+SELECT
+    si. *
```

Comment:
Is this space between `si.` and `*` expected? Seems like it's working with it, but threw me off initially!

Reply:
it works, but it really shouldn't be there. i downloaded a sql formatter extension because this query was originally a large query for multiple things with multiple case statements 🤪 - when it formats, it throws a space in there. i thought it was odd but figured it knows more than me 😂 removed the space! thanks Clay